File: server/zlj2019/game_server/src/com/imop/lj/gameserver/pet/msg/CGPetRefreshTalentSkill.java
package com.imop.lj.gameserver.pet.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.CGMessage;
import com.imop.lj.gameserver.pet.handler.PetHandlerFactory;

/**
 * 宠物刷新天赋技能
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGPetRefreshTalentSkill extends CGMessage{
	
	/** 武将唯一Id */
	private long petId;
	
	public CGPetRefreshTalentSkill (){
	}
	
	public CGPetRefreshTalentSkill (
			long petId ){
			this.petId = petId;
	}
	
	@Override
	protected boolean readImpl() {

	// 武将唯一Id
	long _petId = readLong();
	//end



			this.petId = _petId;
		return true;
	}
	
	@Override
	protected boolean writeImpl() {

	// 武将唯一Id
	writeLong(petId);


		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.CG_PET_REFRESH_TALENT_SKILL;
	}
	
	@Override
	public String getTypeName() {
		return "CG_PET_REFRESH_TALENT_SKILL";
	}

	public long getPetId(){
		return petId;
	}
		
	public void setPetId(long petId){
		this.petId = petId;
	}


	@Override
	public void execute() {
		PetHandlerFactory.getHandler().handlePetRefreshTalentSkill(this.getSession().getPlayer(), this);
	}
}

File: u3u2/Assets/Scripts/app/net/c2s/CGPetRefreshTalentSkill.cs
using System;
using System.IO;
namespace app.net
{

/**
 * 宠物刷新天赋技能
 * 
 * @author CodeGenerator, don't modify this file please.
 */
public class CGPetRefreshTalentSkill :BaseMessage
{
	
	/** 武将唯一Id */
	private long petId;
	
	public CGPetRefreshTalentSkill ()
	{
	}
	
	public CGPetRefreshTalentSkill (
			long petId )
	{
			this.petId = petId;
	}
	
	protected override void ReadImpl() 
	{
		return;
	}
	
	protected override void WriteImpl() 
	{
	// 武将唯一Id
	WriteLong(petId);

	}
	
	public override short GetMessageType()
	{
		return (short)MessageType.CG_PET_REFRESH_TALENT_SKILL;
	}
	
	public override string getEventType()
	{
		return "";
	}
	
	}
}